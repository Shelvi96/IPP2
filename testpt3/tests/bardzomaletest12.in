NEW ID

7537>4406632
125033>174;183
:;5:485>0
7;;21847>8:94;88
9451746>132733
6797752>50;9;7
192::17>6000;786
3>9:04
54;686413>068;;7;;7:
3>0:3:97614
0;::3>6280;7;
62580>55;8708110
5324>6:16
011:;747;>4481;914
@88
@:4482524:7986845;1
@823;3445905229:23887638698368
@482775;;;9343526792390:5:;6
@662:24:92915:496:309165:
@3703561;0;1813:343105;2
@305997;03078:
@0:05:1898
@4:511;25;;6
@761::0:812:580:4171
@9:;:913112180
@::70:0305;07722003:;28599:;
@:;581
@034731:48::3::699856:6414211:
@53:9:1:197;;:
@986;53579:820173
@1:::0;1110042:6:85702214379
@34042:62278
