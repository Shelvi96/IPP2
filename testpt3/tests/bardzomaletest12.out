0
100
45950095345464266
379752854094746
31424165379
25957118088
1
0
0
532170
1
22877179875450
0
45984563824811304
0
1332
1284673584319488
0
